### PR TITLE
Upgrade to support latest versions, remove outdated code, fix tests.

### PR DESCRIPTION
**Pull request by throwable-one**
* No 2.6, 3.5
* Pytest 4, 5 only
* Coverage 4

### CODE REVIEW

File: tests/integration-tests/flake8_test.py [hunk 1]
```diff
@@ -7,38 +7,11 @@
 from service_messages import ServiceMessage, assert_service_messages, parse_service_messages
 
 
-@pytest.fixture(scope='module', params=["flake8==2.0.0", "flake8==2.4.0", "flake8==2.6.0"])
-def venv_flake8_v2(request):
-    return virtual_environments.prepare_virtualenv([request.param])
-
-
-@pytest.fixture(scope='module', params=["flake8==3.0.3"])
+@pytest.fixture(scope='module', params=["flake8==3.7.9"])
```
Comment (shalupov) [inline tests/integration-tests/flake8_test.py]: I think we need to check latest flake8 version anyway. It wan't there before but still

File: tests/integration-tests/pylint_integration_test.py [hunk 2]
```diff
@@ -7,19 +7,19 @@
 
 
 if sys.version_info < (3, ):
-    pylint_versions = ['==1.8', '==1.9.4']
+    pylint_versions = ['==1.9.5']
 else:
-    pylint_versions = ['==1.8', '==1.9.4', '==2.2', '==2.3.1', '==2.4.0', '>2.4']
+    pylint_versions = ['==2.2.3', '==2.4.4']
```
Comment (shalupov) [inline tests/integration-tests/pylint_integration_test.py]: Also the latest version

File: tests/integration-tests/flake8_test.py (hunk 1 above)
Comment (throwable-one) [reply tests/integration-tests/flake8_test.py]: done

File: tests/integration-tests/pylint_integration_test.py (hunk 2 above)
Comment (throwable-one) [reply tests/integration-tests/pylint_integration_test.py]: done

**Pull request merged**